src/renderer/dx/DxRenderer.cpp

```diff
@@ -1674,13 +1679,13 @@ CATCH_RETURN()
 // Return Value:
 // - S_OK or relevant DirectX error
 [[nodiscard]] HRESULT DxEngine::PaintBufferLine(gsl::span<const Cluster> const clusters,
-                                                COORD const coord,
+                                                COORD coord,

```

Comment:
Not sure why `const` disappeared. You're not modifying it as far as I can tell.

Reply:
Much of this PR was sponsored by the Search & Replace Inc. (with regexp). This is probably one of the many oopsies I forgot to revert. 😅

Reply:
Wuh oh.